src/components/templates/PartyMap/components/RoomModal/RoomModal.scss

```diff
@@ -45,6 +38,17 @@ $spacing: 20px;
     }
   }
 
+  &__icon {
+    flex-shrink: 0;
+    height: 180px;
+    width: 180px;
```

Comment:
Should these be `min-width` and `min-height` or similar?

Reply:
No, we tried it and it doesn't actually look better

Reply:
I guess since we set the flex area to never change size it probably doesn't matter. My thoughts here was that this will never responsively resize. But you'd probably have to get down to mobile browser size for that to matter at this stage I would guess, and we likely have a ton of other visual/other issues at that point anyway. 👍🏻